examples/nlp/language_modeling/megatron_t5_seq2seq_eval.py

```diff
@@ -65,9 +65,14 @@ def main(cfg) -> None:
             trainer.callbacks[idx] = StatelessTimer(cfg.trainer.max_time,)
 
     # Get the T5 Base configuration.
-    t5_cfg = MegatronT5GLUEModel.restore_from(
-        restore_path=cfg.model.restore_from_path, trainer=trainer, return_config=True
-    )
+    if hasattr(t5_cfg.data.validation_ds, 'task_name'):
+        t5_cfg = MegatronT5GLUEModel.restore_from(
```

Comment:
Do we need the correct model for restore_from when getting config? Can't we use `ModelPT` for both?

Reply:
I'd rather keep it this way because we don't have a CI/unit test yet for this eval script.